test/lint.js

```diff
@@ -46,42 +46,45 @@ function checkSchema(dataFilename) {
     console.log('   ' + ajv.errorsText(ajv.errors, {
       separator: '\n    ',
       dataVar: 'item'
-    })
-  );
-}
+    }));
+  }
 }
 
 
 function load() {
-  var dirAbs;
+  for (let file of arguments) {
```

Comment:
Not sure it is necessary, `arguments` is iteratable. Can you elaborate, I'm not sure I get your point.